### DISCUSSION TITLE
docker questions

### DISCUSSION THREAD

**Discussion by lephtHanded** [TechnitiumSoftware/DnsServer#422] [Q&A]
Hey, awesome app!
this weekend i'll be moving all my local network A records to this and changing the DNS IP's.

i do have a couple of questions though:
 - How can i set up a couple different hosts and sync their configs?  (i only want to enter all the DNS records on one and have them sync to the other)  (also would like to have 2 DNS IP entries in the DHCP settings in case one of my servers goes kaput)

 - I tried setting the docker option to show local time in the logs, but that didn't work (OR docker time zone is set to Zulu).  I also dug around the settings section of the GUI and couldn't find a time zone selection.  Maybe add that to the settings in the GUI?

 - Themes?  so much bright!

thanks again for an awesome product!

**Comment by ShreyasZare** [comment]
> Hey, awesome app! this weekend i'll be moving all my local network A records to this and changing the DNS IP's.

Thanks for the compliments.

> * How can i set up a couple different hosts and sync their configs?  (i only want to enter all the DNS records on one and have them sync to the other)  (also would like to have 2 DNS IP entries in the DHCP settings in case one of my servers goes kaput)

You can do that by creating a primary zone on the first server and creating a secondary zone on the second server. Any changes that you make in the primary zone will get automatically synced to the secondary zone. 

Note that you will need to add an NS record in the primary zone with the domain name of the secondary zone so that zone transfer is allowed for it. Alternatively, you can edit the primary zone's options and allow zone transfer for the secondary server's IP address.

> * I tried setting the docker option to show local time in the logs, but that didn't work (OR docker time zone is set to Zulu).  I also dug around the settings section of the GUI and couldn't find a time zone selection.  Maybe add that to the settings in the GUI?

You need to configure the time zone for the docker container. Your host's time zone is not used and by default docker container will use UTC time zone.

> * Themes?  so much bright!

There is no plan to add dark mode due to lack of time to implement it.

> thanks again for an awesome product!

You're welcome!
